rwengine/CMakeLists.txt

```diff
@@ -159,3 +159,10 @@ target_include_directories(rwengine
     )
 
 openrw_target_apply_options(TARGET rwengine)
+
+if(INSTALL_LIBS)
+    install(TARGETS rwengine
```

Comment:
Is there a reason we'd want to install the static library?

Reply:
Support for third person total conversions :rofl: .

Shared libraries should be installed.
We can switch the test to `BUILD_SHARED_LIBS`.